### DISCUSSION TITLE
Dynamic Form from JSON or YML file

### DISCUSSION THREAD

**Discussion by NikolaStanisavljevic** [react-hook-form/react-hook-form#3249] [Q&A] [answered]
I was wondering is it possible to achieve with this library, to build form from JSON file, but form needs to be dynamic, for example in first if one radio button is selected few more fields bellow will appear ?

Thanks for your attention, and thanks for all contributors to this great project

**Comment by bluebill1049** [comment, marked as answer]
this? https://github.com/vtex/react-hook-form-jsonschema

**Reply by NikolaStanisavljevic** [reply]
No, but thank you. I did find it at the end .
 https://github.com/rjsf-team/react-jsonschema-form
Schema dependencies was the thing i was looking for in this case.

**Reply by regicsolutions** [reply]
@NikolaStanisavljevic i am looking to do the same thing wondering how you solved it?
